test/compflags/configs/setViaImplicitModuleName.chpl

```diff
@@ -0,0 +1,8 @@
+config param p = 1;
+config const c = 2;
+config var   v = 3;
+config type  t = int;
+
+writeln((p,c,v,t:string));
+
+
```

Comment:
Mildly nit-picky but why are there two extra lines at the end of this and the previous test?